src/components/navBar/navBar.module.scss

```diff
@@ -79,4 +78,106 @@ $theme-active: #87d870;
         color: $white;
         font-weight: 600;
     }
+
+    .hamburgerIcon {
+        display: none;
+    }
+    ul li {
+        list-style: none;
+    }
+    .navLinks ul {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        gap: 1rem;
+    }
+    .active {
+        color: $theme-active;
+        text-decoration: underline;
+        text-decoration-thickness: $thickness;
+        text-underline-offset: $offset;
+    }
+}
+@media (max-width: 970px) {
+    .navBar {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        position: relative;
+        .navLinks {
+            a {
+                padding: 10px 40px;
+                text-align: left;
+                color: $theme-blue;
+                display: none;
+            }
+        }
+    }
+
+    .signInLink {
+        position: absolute;
+        top: 10px;
+        right: 20px;
+    }
+
+    .showLogo {
+        display: none;
+    }
+
+    .hamburgerIcon {
+        display: inline-block !important;
```

Comment:
do remove the !important.

Reply:
!important here at hamburgerIcon class is necessary, if I remove this then the hamburger Icon is getting removed and Navbar is not looking good, rest I have removed important from everywhere. I know important should be avoided as much as possible but here it was needed

Reply:
not a great practice, this is used in cases where we haven't written good CSS, it works as an escape.
Please changes this.

Reply:
Vinayak, we actually need to discuss this, please let me know your availability so that we can connect to make you understand the scenario

Reply:
actually, this is the reason behind to use !important "not a great practice, this is used in cases where we haven't written good CSS, and it works as an escape" because when media query is not applied hamburgericon has display as none but when media query is getting applied  on removing !important same display none property is getting applied which is disrupting navbar so I was not getting options rather than to include !important

Reply:
and for further clarifications, let's connect over VC